aten/src/ATen/native/mps/operations/View.mm

```diff
@@ -733,6 +733,10 @@ static IntArrayRef updateTensorBaseShape(const Tensor& self) {
   const bool srcComplex = dtypeSrc[dtypeSrc.size() - 1] == '2';
   const bool dstComplex = dtypeDst[dtypeDst.size() - 1] == '2';
   if (dstComplex) {
+    // TODO: Document why explicit cast is needed only for bfloat types
```

Comment:
I just copy-n-pasted TODO from another line. I guess I'll need a help from @kulinseth because I could not find anything in the https://developer.apple.com/metal/Metal-Shading-Language-Specification.pdf that talks about it